src/codegen/codegen_cpp_visitor.cpp

```diff
@@ -2455,7 +2450,8 @@ std::string CodegenCppVisitor::get_variable_name(const std::string& name, bool u
 void CodegenCppVisitor::print_backend_info() {
     time_t tr{};
     time(&tr);
-    auto date = std::string(asctime(localtime(&tr)));
+    char timeString[100];
+    std::strftime(timeString, std::size(timeString), "%A %c", std::localtime(&tr));
```

Comment:
The pedantic complaint here is why `char[100]`? (Does it work for all locales, etc.) Could you avoid theses issues by using `fmt`?

Reply:
True this could be improved. Instead of the suggestion from `cppcheck` I used a function from `chrono`